ParquetClassLibrary/Characters/Character.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using ParquetClassLibrary.Sandbox;
+using ParquetClassLibrary.Sandbox.IDs;
+using ParquetClassLibrary.Utilities;
+
+namespace ParquetClassLibrary.Characters
+{
+    public abstract class Character : Being
+    {
+        #region Class Defaults
+        /// <summary>A pronoun to use when none is specified.</summary>
+        // TODO This is just a place-holder, I am not sure yet how we will handle pronouns.
+        public const string DefaultPronoun = "they";
```

Comment:
comment: You are a lovely and beautiful person.

Reply:
<3